src/components/Pressable/Pressable.tsx

```diff
@@ -56,12 +56,19 @@ export default function Pressable(props: PressableProps) {
     [isPressCallbackEnabled, props.onLongPress, isPressedDown]
   );
 
+  // clear timeout on exit,
```

Comment:
```suggestion
  // clear timeout on exit
```

But I think that "on exit" doesn't fit well here 😅 Especially that `hoverOutTimeout` is cleared on "enter"

Reply:
updated to: `// cancel timeout when the other one is started`. I'm not sure if maybe it'd be better to delete it all together? What do you think?

Reply:
Well, I don't think this comment is necessary. On the other hand it would be nice to have it (cc @j-piasecki).

```
  // cancel timeout when the other one is started
```

I think it is more of a side effect.

<details>
<summary> If you take a look at the code  </summary>

```tsx
const hoverInTimeout = useRef<number | null>(null);
const hoverOutTimeout = useRef<number | null>(null);

const hoverGesture = useMemo(
  () =>
    Gesture.Hover()
      .onBegin((event) => {
        if (hoverOutTimeout.current) {
          clearTimeout(hoverOutTimeout.current);
        }
        if (props.delayHoverIn) {
          hoverInTimeout.current = setTimeout(
            () => props.onHoverIn?.(adaptStateChangeEvent(event)),
            props.delayHoverIn
          );
          return;
        }
        props.onHoverIn?.(adaptStateChangeEvent(event));
      })
```

</details>

`hoverOutTimeout` is cleared inside `onBegin` callback and there's no check if `hoverInTimeout` is started, so that comment is a bit misleading (at least for me).

Reply:
I agree, I think unlike the other comments which clear up non-oblivious mechanics, this one is just unnecessary.
I'll remove it for now, with possibility of adding it back later.